scripts/in_container/_in_container_utils.sh

```diff
@@ -259,11 +259,15 @@ function install_released_airflow_version() {
         export SLUGIFY_USES_TEXT_UNIDECODE=yes
     fi
     rm -rf "${AIRFLOW_SOURCES}"/*.egg-info
-    INSTALLS=("apache-airflow==${1}" "werkzeug<1.0.0")
-    pip install --upgrade "${INSTALLS[@]}"
+    if [[ ${INSTALL_AIRFLOW_VERSION} == "wheel" ]]; then
+        pip install /dist/apache_airflow-*.whl
+    else
+        INSTALLS=("apache-airflow==${1}" "werkzeug<1.0.0")
```

Comment:
?

Reply:
It's there for months to install old 1.10 versions. We do not need it any more, I guess. I will try it.